stripe/src/main/java/com/stripe/android/model/Address.kt

```diff
@@ -43,10 +43,15 @@ data class Address internal constructor(
             this.city = city
         }
 
+        @Deprecated("This will be removed in future versions.")
         fun setCountry(country: String?): Builder = apply {
             this.country = country?.toUpperCase(Locale.ROOT)
         }
 
+        fun setCountry(countryCode: CountryCode?): Builder = apply {
+            this.country = countryCode?.twoLetters
+        }
```

Comment:
I found a case where internally we called setCountry and used the string "United States" instead of the country code.  So having a strongly typed value here seemed like a good idea.

I think if we don't want to expose it, we should just skip it in Address and let it be a String.

I would not be surprised if others are using something other than the two-letter country code, but maybe it doesn't matter?  The API [here](https://stripe.com/docs/api/tokens/create_card) doesn't really specify

Reply:
I'd be in favor of leaving `Address` as-is and making it an internal-only thing for now.

Reply:
So you are saying have a setCountry in the builder that is internal only, and leave the existing setCountry which continues to take a string and is not deprecated?

Reply:
Yeah, I think that would also work.

Reply:
I removed to the changes to the Address class for now.  It was easier for the moment.